app/main.py

```diff
@@ -1,8 +1,30 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        # when obj created add link on it to the dict "people"
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons = []
+    for human in people:
+        person = Person(name=human["name"], age=human["age"])
+        persons.append(person)
+        # temporary set attributes "husband" and "wife" if they exist
+        if "wife" in human.keys() and human["wife"] is not None:
+            person.wife = human["wife"]     # set name as string
+        elif "husband" in human.keys() and human["husband"] is not None:
+            person.husband = human["husband"]       # set name as string
```

Comment:
You can simplify the check by using the .get() dictionary method. If the dictionary does not have such a key, it will return None.